repository-data/webfiles/src/main/resources/site/src/scss/components/_article-header.scss

```diff
@@ -117,6 +117,13 @@
     @include core-font(24);
 }
 
+.article-header__types {
+    &--push {
+        display: block;
+        margin-top: 11px;
+    }
+}
+
```

Comment:
This change may be fine. However can you check if `nhsd-!t-margin-10` (from the toolkit) would work? Otherwise, it's okay.

Reply:
I am afraid our knowledge of the UI is very limited and tackling this may delay the release. Is it ok if we do not fix this now?